src/components/LineChart.tsx

```diff
@@ -61,25 +67,41 @@ function LineChart({
       className={`${className} ${
         rotate
           ? "rotate-90 h-[390px] w-[90vh] bg-inherit ml-[-50vw] mt-[22vh]"
-          : "relative ml-[5vw] sm:ml-[1vw] mr-[5vw] sm:mr-[0] md:mr-[0] sm:h-[340px] bg-[#22262C]"
+          : "relative ml-[5vw] sm:ml-[1rem] mr-[5vw] sm:mr-[0] md:mr-[0] sm:h-[340px] bg-[#22262C]"
       }`}
     >
       {width < 700 &&
         (location.pathname.includes("opm") ||
           location.pathname.includes("opmcomparison")) && (
           <div className="flex items-center justify-between ml-[20px] mr-[20px] pt-[16px]">
             <p className="text-white">{title}</p>
-            <div className="bg-[#383F47] w-[30px] h-[30px] rounded-full">
-              <CustomImage
-                src={RotateIcon}
-                className="relative top-[8px] left-[9px]"
-                alt="Filter Icon"
-                onClick={onRotateHandler}
+            <div className="flex items-center">
+              <CustomTab
+                className="opm-tabs mr-4"
+                tabData={CHART_TABS}
+                tabValue={tabValue}
+                setTabValue={setTabValue}
               />
+              <div className="bg-[#383F47] w-[30px] h-[30px] rounded-full">
+                <CustomImage
+                  src={RotateIcon}
+                  className="relative top-[8px] left-[9px]"
+                  alt="Filter Icon"
+                  onClick={onRotateHandler}
+                />
+              </div>
             </div>
           </div>
         )}
-      <Line options={options} data={data} />
+      {tabValue === 1 ? (
```

Comment:
LineChart is supposed to be re-usable.
That is why I keep linechart.tsx separate from opm.tsx and opmcomparison.tsx;

But after these changes, it looks like there is no way to use Linechart.tsx without including a custom tab for Barchart.

Please find a workaround so that LineChart.tsx is just for Line charts and can be re-used.

Reply:
Removed Barchart from the component and created a new Barchart comp